### PR TITLE
Update privacy text to match desktop

### PR DESCRIPTION
**Pull request by schwers**
https://reddit.atlassian.net/browse/MOBILEWEB-270

Updates privacy test to match desktop

image

@ajacksified

### CODE REVIEW

File: src/views/pages/register.jsx [hunk 1]
```diff
@@ -99,12 +99,9 @@ class RegisterPage extends BasePage {
...
           <div className='text-muted text-small'>
-            We care about your privacy, and we never spam. By creating an
-            account, you agree to reddit's
-            <a href='http://reddit.com/help/useragreement' className='text-link'> User Agreement </a>
-            and
-            <a href='http://reddit.com/help/privacypolicy' className='text-link'> Privacy Policy</a>.
-            We're proud of them, and you should read them.
+            By signing up, you agree to our <a href='https://reddit.com/help/useragreement' className='text-link'>Terms </a>
+            and that you have read our <a href='https://reddit.com/help/privacypolicy' className='text-link'>Privacy Policy </a>
+            and <a href='https://www.reddit.com/help/contentpolicy/' className='text-link'>Content Policy</a>.
```
Comment (ajacksified) [inline src/views/pages/register.jsx]: These should probably open up in a separate tab/window, so that you don't wipe out your registration info.

**Comment by ajacksified** [comment]
:nail_care:

**Comment by schwers** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish: :tada:

**Pull request merged**